src/components/Form.js

```diff
@@ -31,29 +31,34 @@ function Form() {
   };
 
   return (
-    <form onSubmit={bookState}>
+    <form onSubmit={bookState} className="ps-5 pe-5 pb-5 mb-5">
+      <hr />
+      <h2 className="form-header">Add New Book</h2>
       <input
+        className="inputs"
         type="text"
         onChange={changeState}
         value={formValue.author}
         placeholder="Author"
         name="author"
       />
       <input
+        className="inputs"
         type="text"
         onChange={changeState}
         value={formValue.title}
         placeholder="title"
         name="title"
       />
       <input
+        className="inputs"
         type="text"
         name="category"
         value={formValue.category}
         onChange={changeState}
         placeholder="category"
       />
-      <input type="submit" value="Add Book" />
+      <input type="submit" value="Add Book" className="btn btn-primary" />
     </form>
```

Comment:
- [ ] Kindly fix the styling for this section it should look like the design provided, like this:
![image](https://user-images.githubusercontent.com/20567503/185748151-18480de3-54f8-44a4-a2f9-a552738d7d07.png)